### DISCUSSION TITLE
Theme Classic in 2.0.6 version with PS 8.0.4

### DISCUSSION THREAD

**Discussion by jlores** [PrestaShop/PrestaShop#32587] [Q&A] [answered]
Hello,

I always update prestashop with Autoupgrade module and check the update theme. Now in version 8.0.4 I have the theme classic with version 2.0.6 when it should be 2.0.8

Is this a bug of Autoupgrade? Anyone else has happened?

Is there a way to update the theme manually?

Many thanks

**Comment by kpodemski** [comment, marked as answer]
Hello

Sources are here:
https://github.com/PrestaShop/classic-theme/releases/tag/2.0.8

You can also see the difference between 2.0.6 and 2.0.8:
https://github.com/PrestaShop/classic-theme/compare/2.0.6...2.0.8

**Comment by jlores** [comment]
Hello,


I have searched, but I can't find anywhere how to pass the sources to an installable theme. If I take the sources file it is not installable. gives error configuration file missing

Thanks

**Comment by jlores** [comment]
Hello,

Compilation completed :)

Many thanks

**Comment by jlores** [comment]
![Captura desde 2023-05-16 19-14-23](https://github.com/PrestaShop/PrestaShop/assets/48771938/242addaf-4f35-4116-bbf4-aebb9efc50bb)

**Comment by kpodemski** [comment]
Nice. So you managed to update the theme successfully @jlores ?

**Comment by jlores** [comment]
Hello,

I managed to update the theme, compiling 2.0.8, but the checkout with php > 7.4 still doesn't work. I have to continue investigating why

It has to be something from the shipping module. I will wait to see if they release a new version and fix it.

Many thanks for your work
